<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Don't have an account yet? 
 Click below!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(130,130,130);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="loginEdit">
     <property name="styleSheet">
      <string notr="true">QLineEdit{
font-size:14px;
border-radius:6px;
background-color:rgb(110,110,110);
padding-top:6px;
padding-bottom:6px;
}
</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="pwdEdit">
     <property name="styleSheet">
      <string notr="true">QLineEdit{
font-size:14px;
border-radius:6px;
background-color:rgb(110,110,110);
padding-top:6px;
padding-bottom:6px;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="loginButton">
     <property name="styleSheet">
      <string notr="true">QPushButton{
font-size:14px;
border-radius:6px;
background-color:rgb(110,110,110);
padding-top:6px;
padding-bottom:6px;
}

QPushButton:hover{
background-color:rgb(130,130,130);
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="registerButton">
     <property name="styleSheet">
      <string notr="true">QPushButton{
font-size:14px;
border-radius:6px;
background-color:rgb(110,110,110);
padding-top:6px;
padding-bottom:6px;
}

QPushButton:hover{
background-color:rgb(130,130,130);
}</string>
     </property>
     <property name="text">
      <string>Sign up</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
